models/dbHelpers.js

```diff
@@ -0,0 +1,47 @@
+const knexConfig = require('../knexfile')
+const knex = require('knex')
+
+const db = knex(knexConfig.development)
+const model = 'dishes'
+
+
+const getDishes = () => {
+
+    return db(model)
+}
+
+
+
+const getDish = id => {
+    if(id) {
+        return db(model)
+        .where({ id: id})
+        .first();
+    }
+}
+
+const addDish = dish => {
+    const id = db(model).insert(dish);
+    return id
+}
+
+const getRecipes = () => {
+
+    return db('recipes')
+}
+
+const addRecipe = recipe => {
+    const id = db(model).insert(recipe);
```

Comment:
Make sure you are handling the promise appropriately. You should use async/await or .then.